packages/shared/src/components/cards/Card.tsx

```diff
@@ -43,7 +43,7 @@ export const ListCardTitle = classed(Title, 'mr-2');
 
 export const CardTextContainer = classed('div', 'flex flex-col mx-4');
 
-export const CardImage = classed(Image, 'rounded-12 h-40');
+export const CardImage = classed(Image, 'rounded-12 h-40 object-cover');
```

Comment:
Since this now serves many cases just make sure to give it a proper thorough check across implementations  🙏

Reply:
Sure.